experimental/federated_plugin/federated_plugin/example/README.md

```diff
@@ -0,0 +1,16 @@
+# federated_plugin_example
+
+Demonstrates how to use the federated_plugin plugin.
+
+## Getting Started
```

Comment:
If you're going to keep this file, I'd recommend cutting everything after line 3.

Reply:
I have kept this file and removed the federated_plugin/federated_plugin/README